src/libs/actions/Report.js

```diff
@@ -1680,6 +1680,18 @@ function setLastOpenedPublicRoom(reportID) {
     Onyx.set(ONYXKEYS.LAST_OPENED_PUBLIC_ROOM_ID, reportID);
 }
 
+/**
+ * Navigates to the last opened public room
+ *
+ * @param {String} lastOpenedPublicRoomID
+ */
+function navigateToLastOpenedPublicRoom(lastOpenedPublicRoomID) {
```

Comment:
Maybe we could name it this way since it lives in the Report actions file.

```suggestion
function openLastOpenedPublicRoom(lastOpenedPublicRoomID) {
```